backend/codesirius/problems/serializers/execution_constraint_v2.py

```
@@ -0,0 +1,203 @@
+
+from problems.models import ExecutionConstraint
+
+logger = logging.getLogger(__name__)
+
+
+class BulkOperationError(APIException):
+    status_code = 400
+    default_detail = "Bulk operation failed"
+    default_code = "bulk_operation_failed"
+
+    def __init__(self, errors):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)